html + css exercise/styles/style.scss

```diff
@@ -26,90 +34,89 @@ body {
 header {
     position: fixed;
     top: 0;
-    width: 100%;
+    width: 100vw;
     font-size: 100%;
     text-align: center;
     background-color: $primary-color;
     color: rgba(255, 255, 255, 0.95);
     padding: 0.8rem;
-    box-shadow: 1px 1px 15px -3px rgba(0, 0, 0, 0.5); 
+    box-shadow: $box-shadow;
 }
 
 .table-container {
     border-radius: 4px;
     padding: 20px;
     background-color: white;
-    box-shadow: 1px 1px 15px -3px rgba(0, 0, 0, 0.5);
+    box-shadow: $box-shadow;
     margin-bottom: 100px;
-    max-height: 400px; /* Control how many data rows are seen at a time */
+    max-height: $max-table-rows-height;
 }
 
 .user-info-container {
     margin-top: 120px; 
     margin-bottom: 90px;
-}
 
-.user-info-container h2 {
-    font-size: 30px;
-    color:$primary-color;
-    text-align: center;
-    margin-bottom: 20px;
+    h2 {
+        font-size: 30px;
+        color: $primary-color;
+        text-align: center;
+        margin-bottom: 20px;
+    }
 }
 
 .user-info-table {
```

Comment:
Should have I just make the white background of the table as a padding ? This way also making left right padding smaller then up bottom as seen in the exercise attached image?